### DISCUSSION TITLE
School isn't listed on the website.

### DISCUSSION THREAD

**Discussion by JadeBetty** [community/community#86760] [GitHub Education]
### Hi everyone,

Hey there, 
I've tried to go for Github Education but I could not find my school in the map.

**Comment by anuragjain-git** [comment]
You do not need a Student ID to get the GitHub Student Developer Pack, for more information see [How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)

As for your school, you can have an administrator sign up by making them fill out the [GitHub Campus Program Application](https://education.github.com/partner_school_applications/apply)

**Reply by JadeBetty** [reply]
Hey there,

My school doesn't want to complete the Campus program.

**Comment by naveen-98** [comment]
Hello! Try following these first

Double-Check School Name:
Ensure that you're entering the correct name of your school. Sometimes, schools may have variations in their official names, and abbreviations, or there might be a slight difference in the spelling.

Contact GitHub Support:
If you've double-checked and still can't find your school, you can reach out to GitHub Support for assistance. They might be able to manually add your school to the system or provide guidance on how to proceed.

Check Eligibility:
Make sure that your school is eligible for the GitHub Education benefits. GitHub Education is generally available to students at recognized educational institutions. If your school is not recognized, you may need to verify its eligibility or explore alternative options if available.

Alternative Email Domain:
Some schools might use alternative email domains. If your school email domain is not recognized, check with GitHub Support to see if they can assist you in such cases.

Alternative Verification Methods:
GitHub Education might offer alternative verification methods, such as using documents or other means to verify your student status. Check with GitHub Support for guidance on alternative verification processes.
